chapter2_mdw/ExerciseFive

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use 5.010;
+use warnings;
+
+chomp($string=<STDIN>);
+chomp($number=<STDIN>);
+
+$result = $string x $number;
+
+say $result;
```

Comment:
- As pointed for exercise-4, some prompt text and validation would be good.
- This will print the concatenated string in the same line. I thought we had to "prints out the string the number of times indicated by the number on separate lines". I might be mistaken.